.circleci/config.yml

```diff
@@ -20,6 +20,30 @@ jobs:
     parallelism: 4
     steps:
       - checkout
+
+      # Download and cache Kubebuilder.
+      - run:
```

Comment:
can we separate out these steps to run as integration test job instead of overwriting the existing workflow?

Reply:
It's possible, but would require a bigger change on the structure of the workflows. Don't mind doing it later.